docs/features/techdocs/configuring-ci-cd.md

```diff
@@ -0,0 +1,87 @@
+---
+id: configuring-ci-cd
+title: Configuring CI/CD to generate and publish TechDocs sites
+description:
+  Configuring CI/CD to generate and publish TechDocs sites to cloud storage
```

Comment:
Note, Docusaurus doesn't support "real" YAML Frontmatter, and so this may not work. (i previously tried | and > and none built properly) Suggest move it to the line above, and just add a prettier ignore  above the description so it won't wrap. 🤕